articles/azure-monitor/app/opencensus-python.md

```diff
@@ -46,160 +45,113 @@ First you have to create an Application Insights resource which will generate an
 
 2. Click **Create**.
 
-## Configure local forwarder
+## Install OpenCensus Azure Monitor Exporters
 
-1. Select **Overview** > **Essentials** > Copy your application's **Instrumentation Key**.
-
-   ![Screenshot of instrumentation key](./media/opencensus-python/0003-instrumentation-key.png)
-
-2. Edit your `LocalForwarder.config` file and add your instrumentation key. If you followed the instructions in the [pre-requisite](./../../azure-monitor/app/opencensus-local-forwarder.md) the file is located at `C:\LF-WindowsServiceHost`
-
-    ```xml
-      <OpenCensusToApplicationInsights>
-        <!--
-          Instrumentation key to track telemetry to.
-          -->
-        <InstrumentationKey>{enter-instrumentation-key}</InstrumentationKey>
-      </OpenCensusToApplicationInsights>
-    
-      <!-- Describes aspects of processing Application Insights telemetry-->
-      <ApplicationInsights>
-        <LiveMetricsStreamInstrumentationKey>{enter-instrumentation-key}</LiveMetricsStreamInstrumentationKey>
-      </ApplicationInsights>
-    </LocalForwarderConfiguration>
-    ```
-
-3. Restart the application **Local Forwarder** service.
-
-## OpenCensus Python package
-
-1. Install the Open Census package for Python and exporter with pip or pipenv from the command line:
+1. Install the OpenCensus Azure Monitor Exporters:
 
     ```console
-    python -m pip install opencensus
-    python -m pip install opencensus-ext-ocagent
-
-    # pip env install opencensus
+    python -m pip install opencensus-ext-azure
     ```
 
     > [!NOTE]
-    > `python -m pip install opencensus` assumes that you have a PATH environment variable set for your Python installation. If you have not configured this, you would need to give the full directory path to where your Python executable is located which would result in a command like: `C:\Users\Administrator\AppData\Local\Programs\Python\Python37-32\python.exe -m pip install opencensus`.
+    > `python -m pip install opencensus-ext-azure` assumes that you have a PATH environment variable set for your Python installation. If you have not configured this, you would need to give the full directory path to where your Python executable is located which would result in a command like: `C:\Users\Administrator\AppData\Local\Programs\Python\Python37-32\python.exe -m pip install opencensus-ext-azure`.
 
 2. First let's generate some trace data locally. In Python IDLE, or your editor of choice, enter the following code:
```

Comment:
There is a "First" here too, even though it is the second step. A little confusing.